website/content/docs/k8s/installation/deployment-configurations/clients-outside-kubernetes.mdx

```diff
@@ -35,14 +48,69 @@ annotation `consul.hashicorp.com/auto-join-port` to an integer value or
 a named port to specify the port for the auto-join to return. This enables
 different pods to have different exposed ports.
 
-## Networking
+### Auto-join on the Pod network
+In the default Consul Helm chart installation, the Consul clients and servers
+are deployed as pods and routeable only via their `podIP:containerPort` for
+gossip/RPC. This would mean any external client agents joining would need to
+be able to have connectivity to those pod IPs.
+
+Given you have the [official Helm chart](/docs/k8s/helm) installed with the default values, do the following to join an external client agent. 
+
+1. Make sure the pod IPs of the clients and servers in Kubernetes are routeable from the VM and that the VM IP (`$ADVERTISE_IP`) is routeable from the pods, adding any firewall rules necessary to allow traffic on the gossip and RPC ports.
 
-Consul typically requires a fully connected network.
-Because the Consul Helm chart currently doesn't allow exposing servers' gossip ports via a `hostPort`,
-nodes outside of Kubernetes joining a cluster running within Kubernetes must be able to communicate
-to pod IPs via the network. Note that the auto-join provider discussed above will use pod IPs by default.
+2. Make sure you have the `kubeconfig` file for the Kubernetes cluster in `$HOME/.kube/config`.
+
+2.  On the external VM, run: 
+```shell-session
+consul agent \
+  -advertise="$ADVERTISE_IP" \
+  -retry-join='provider=k8s label_selector="app=consul,component=server"'
+  -bind=0.0.0.0 \
+  -client=0.0.0.0 \
+  -hcl='leave_on_terminate = true' \
+  -hcl='ports { grpc = 8502 }' \
+  -config-dir=$CONFIG_DIR \
+  -datacenter=$DATACENTER \
+  -data-dir=$DATA_DIR \
+```
+
+3. Check if the join was successful by running `consul members`
+
+### Auto-join on the Host network
```

Comment:
I think since host network and host ports are two separate concepts in Kubernetes, it's worth being precise here since using host network has security implications.

```suggestion
### Auto-join via host ports
```